src/app/components/core/developer/additional-services/additional-services-utils.service.ts

```diff
@@ -69,12 +69,8 @@ export class AdditionalServicesUtils {
     private alertService: AlertService) {
   }
   public getAllServiceStatus(autoRefresh: boolean, from: string) {
-    // There are two more additional services exist in the FogLAMP. Get the current service name (Fledge/FogLAMP) and show resepected services detail accordingly
-    const currentService = JSON.parse(sessionStorage.getItem('SERVICE_NAME'));
-    let expectedExternalServiceType = ['Notification', 'Dispatcher'];
-    if (currentService === 'FogLAMP') {
-      expectedExternalServiceType = ['Notification', 'Management', 'Dispatcher', 'BucketStorage'];
-    }
+    // FYI; Four additional services exist in the Fledge and two in FogLAMP. There will be some extra API calls in Fledge (to search all of the expected additional services in different API response) because we don't have any info whether user is FogLAMP or Fledge.
```

Comment:
// NOTE: 

BTW, which `Four additional services exist in the Fledge`?

Reply:
Good catch!
I'll fix this typo. Two services for Fledge and four for FogLAMP.